src/test/Accounting.t.sol

```diff
@@ -613,4 +613,88 @@ contract AccountingTest is Setup {
 
         assertEq(asset.balanceOf(address(yieldSource)), _amount);
     }
+
+    function test_deposit_zeroAssetsPositiveSupply_reverts(
+        address _address,
+        uint256 _amount
+    ) public {
+        _amount = bound(_amount, minFuzzAmount, maxFuzzAmount);
+        vm.assume(
+            _address != address(0) &&
+                _address != address(strategy) &&
+                _address != address(yieldSource)
+        );
+
+        setFees(0, 0);
+        mintAndDepositIntoStrategy(strategy, _address, _amount);
+
+        uint256 toLoose = _amount;
```

Comment:
Did you mean `toLose`?  The amount that will be lost -> amount to lose?

Reply:
https://github.com/yearn/tokenized-strategy/pull/103/commits/af38ff5249f0013e02c6acfaab3abf32e13b1abe